src/backend/utils/misc/guc.c

```diff
@@ -2874,6 +2875,18 @@ static struct config_string ConfigureNamesString[] =
 		check_temp_tablespaces, assign_temp_tablespaces, NULL
 	},
 
+	{
+		{"temp_file_tablespaces", PGC_USERSET, CLIENT_CONN_STATEMENT,
```

Comment:
how about to name new feature "temp_filespaces"? because it is for some **space** for **temp** **file**s, like temp_tablespaces is some **space** for **temp** **table**s